app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index; end
+
+  def edit; end
+
+  def update
+    current_user.update(user_params)
+    if current_user.valid?
+      flash[:success] = "Successfully Updated Your Profile"
```

Comment:
Can we go with a sentence-casing instead of title-casing?

Reply:
Good idea